docs/UserGuide.md

```diff
@@ -128,9 +128,12 @@ This section describes the available features in TAB.
 * If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
   e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `calendar`, `exit`, `clear` etc.) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
```

Comment:
![image](https://user-images.githubusercontent.com/76084739/139630573-b90d19ff-3ba6-48aa-acbe-74eb26f1c231.png)

![image](https://user-images.githubusercontent.com/76084739/139630609-1e69c331-8776-45f8-a0c1-b98a15f21e28.png)
both `help 123` and `remind 123` are invalid commands and returns the same result showing valid command for help, same for other single word commands